### DISCUSSION TITLE
"Approved but no benefits received"

### DISCUSSION THREAD

**Discussion by sqkstt7777** [community/community#146101] [GitHub Education]
### Hi everyone,

![QQ20241203-124926](https://github.com/user-attachments/assets/a1d1f9d8-9d65-49fb-9ff3-181d20a4ccef)
"It has been three days, and I have not received the benefits yet."

**Comment by quoc10973** [comment]
It can over 3 days, you need to wait a little bit to get result

**Comment by davevad93** [comment]
Hi @sqkstt7777 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sqkstt7777 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
